docs/src/examples/components/Grid/Variations/GridExampleKeyboardNavigable.shorthand.tsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react'
+import { Grid, Image, Button, gridBehavior } from '@stardust-ui/react'
+import _ from 'lodash'
+
+const images = [
```

Comment:
can you make this an array to save space?

Reply:
yep, sure, that does make sense. Thank you!